docs/installation-ios.md

```diff
@@ -159,3 +159,68 @@ didReceiveNotificationResponse:(UNNotificationResponse *)response
 If you're having problems with messages not being received, check out the following blog post for help:
 
 https://firebase.googleblog.com/2017/01/debugging-firebase-cloud-messaging-on.html
+
+
+
+## 5) Dynamic Links (optional)
```

Comment:
You should reference `https://firebase.google.com/docs/dynamic-links/ios/receive`.
Something like:
"Make sure to setup dynamic links for iOS as described here..."